src/cljs/main/broadfcui/page/billing/create_project.cljs

```diff
@@ -29,7 +29,9 @@
               [comps/ErrorViewer {:error (:details error)}])
             :else
             (if (empty? billing-accounts)
-              [:div {} "You do not have any billing accounts available."]
+              [:div {} "You do not have any billing accounts available. To learn how to create a billing account, click "
+                [:a {:target "_blank"
+                     :href "https://software.broadinstitute.org/firecloud/guide/topic?name=firecloud-google"} "here"] "."]
```

Comment:
Bigger click targets are better; consider making both words of "click here" the link.